src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js

```diff
@@ -5,17 +5,21 @@ import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
 const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
   className: 'password-authenticator',
   title () {
-    const daysToExpiry = this.options.appState.get('currentAuthenticator').settings.daysToExpiry;
-    return `${loc('password.expiring.title', 'login', [ daysToExpiry ])}`;
+    const passwordPolicy = this.getPasswordPolicy() || {};
+    const daysToExpiry = passwordPolicy.daysToExpiry;
+
+    if (daysToExpiry > 0) {
+      return loc('password.expiring.title', 'login', [daysToExpiry]);
+    } else if (daysToExpiry === 0) {
+      return loc('password.expiring.today', 'login');
+    } else {
+      return loc('password.expiring.soon', 'login');
+    }
   },
 
   save () {
     return loc('password.expired.submit', 'login');
   },
-
-  getPasswordPolicy () {
```

Comment:
The reason why I left it there is because if in `EnrollAuthenticatorPaswordView` the way we retrieve password policy changes, it will break here..

Reply:
I hope they'll stay in sync in terms of how to retrieve policy for Enroll/Reset password.
If changes happen to Enroll but not reset, let it break which remind us to chat with backend to see why the inconsistency. Otherwise it's a bit confusing that seems like we're anticipating there will be inconsistency happening.?